src/Errors/EmptyNameError.ts

```diff
@@ -0,0 +1,8 @@
+export class EmptyNameError extends Error {
+  public static message: string = "name has no length";
```

Comment:
aca pondria un error más generico, uno tipo EmptyStringError, tipo "field can't be empty", porque me suena a boilerplate tener un error de string vacio por cada campo de string que haya en el dominio

Reply:
Además, pondría en los errores directamente

  constructor() {
    super("Field can't be empty");
  }

(y consideraría dar mayúsculas a los mensajes de error, por prolijidad)